h/feeds/rss.py

```diff
@@ -73,8 +71,7 @@ def feed_from_annotations(annotations, annotation_url, rss_url, html_url,
             for annotation in annotations]
     }
 
-    if annotations:
-        feed['pubDate'] = parser.parse(annotations[0]['updated']).strftime(
-            '%a, %d %b %Y %H:%M:%S %Z')
+    if annotations and annotations[0].updated:
```

Comment:
This is mostly for easier testing, so we don't have to always have an object that returns a datetime for `updated` and `created`. I can remove it though if you feel strongly about it.